### ISSUE TITLE
Graphics that might be useful to openage

### ISSUE THREAD

**Issue by roybaer** [SFTtech/openage-data#17]
Hi there!

Some time ago (~12 years) I myself wanted to create a game similar to Age of Empires.
I was able to create some nice graphics and mock-up engines in Turbo Pascal and JavaScript but the game never got anywhere and after all this time probably will never get anywhere.

Are you interested in some graphics files?

![screenshot_20180505_223413](https://user-images.githubusercontent.com/7901580/39667369-b36a668c-50b4-11e8-9e39-486061c7bd5d.png)

As you can see, the overall appearance of the graphics is kind of AoE-esque.  They would therefore be quite a lot better than generic placeholders.

CC-BY-SA licensing would be fine with me.  Let me know what you think.

Side note:  I am also writing an engine clone, namely for Anno 1602:
You can find it here: https://github.com/roybaer/mdcii-engine and I describe it in more detail in the [AnnoZone forum](http://www.annozone.de/forum/index.php?page=Thread&threadID=15456&s=654212a55ab1007de04c176d0f55a1397ab48044) (German).

**Comment by heinezen** [comment]
Hello there!

This looks awesome. I definitely think we can make use of your work, although some additional work in GIMP/Photoshop might be required to match the exact style of AoE. But nevertheless, the addition of these would be welcome :)

Do you still have the 3D model sources available or just the sprites?

**Comment by roybaer** [comment]
Yes, I still have the 3D models, they are in Caligary TrueSpace 2 (*.cob) format, though.
They also make heavy use of that program's procedural textures and Blender's cob importer seems not to like them.

Alright. I'll try to sensibly organize everything in a git repository and report back when I'm done.

**Comment by roybaer** [comment]
This is the preliminary git import: https://github.com/roybaer/Iso_3D_Experiments
It still needs some refactoring and documentation but you can already take a closer look at it.

**Comment by Grubben** [comment]
Any updates on these assets?

**Comment by heinezen** [comment]
@Grubben No, no updates yet.

**Comment by roybaer** [comment]
Realistically speaking, this artwork can only be a stopgap for OpenAge.
It is certainly better than pure "programmer artwork", but re-rendering is not really feasible.
